src/localization/en.json

```diff
@@ -250,5 +250,17 @@
     "answerLabel": "Insert Answer",
     "addAnswer": "+ Add new answer option",
     "saveAnswer": "Save question"
+  },
+  "questionnaireTable": {
+    "title": "Questionnaire Title",
+    "description": "Description",
+    "actions": "Actions",
+    "edit": "Edit",
+    "delete": "Delete",
+    "status": "Status",
+    "confirmDeleteTitle": "Are you sure to delete?",
```

Comment:
```suggestion
    "confirmDeleteTitle": "Are you sure you want to delete?",
```

Reply:
Bit wordy, probably confirm questionnaire delete is good for the title, and the button text can be delete